overrides/inspec/resource_override.rb

```diff
@@ -22,6 +22,8 @@ def self.attributes
         %i[
           manual
           additional_functions
+          product_url
+          privileged
```

Comment:
Can you add a comment for what priviledged means? The rest seem self-explanatory.